mycroft/client/enclosure/eyes.py

```diff
@@ -92,6 +94,13 @@ def volume(self, event=None):
     def reset(self, event=None):
         self.writer.write("eyes.reset")
 
+    def system_reset(self, event=None):
```

Comment:
@penrods we have no guarantee eyes and mouth will be reset at this point because system.reset may restart arduino first. Then, we probably should move all logic regarding the reset of mycroft components to inside arduino and reset them before arduino itself.